main/pipeline/pipeline.cc

```diff
@@ -312,6 +312,9 @@ vector<ast::ParsedFile> incrementalResolve(core::GlobalState &gs, vector<ast::Pa
             // Cancellation cannot occur during incremental namer.
             ENFORCE(result.hasResult());
             what = move(result.result());
+            if (opts.stopAfterPhase == options::Phase::NAMER) {
```

Comment:
We set `stopAfterPhase` in autogen tests. Not respecting it in incremental mode causes new issues now that we aren't (accidentally) recomputing mixins.

Reply:
can we add a comment in source for this?

Reply:
Sure. And as for testing autogen more specifically.... we hardcode it into the test harness right now so it stops after certain phases. I'm not in a mood to change that.

Reply:
Comment added.